static/src/javascripts/projects/common/modules/article/open-module.js

```diff
@@ -31,23 +33,21 @@ define([
     return {
         init: function () {
             if (config.page.openModule) {
-                spacefinder.getParaWithSpace(getSpacefinderRules()).then(function (space) {
-                    if (space) {
-                        ajax({
-                            url: config.page.openModule,
-                            crossOrigin: true,
-                            method: 'get'
-                        }).then(function (resp) {
-                            if (resp.html) {
-                                fastdom.write(function () {
-                                    $.create(resp.html)
-                                        .addClass('element--supporting')
-                                        .insertBefore(space);
-                                    $('.submeta-container--break').removeClass('submeta-container--break');
-                                });
-                            }
-                        });
-                    }
+                spaceFiller.fillSpace(getSpacefinderRules(), function (spaces) {
+                    ajax({
+                        url: config.page.openModule,
```

Comment:
It would be nice if we could make the writer synchronous, otherwise we run the risk of race conditions - we have no guarantee the found space will still exist when the AJAX response returns. I'm not sure if we want to resolve that (existing) issue here and now, though...